### PR TITLE
fix for avatar unable to jump into hover on mac

### PR DESCRIPTION
**Pull request by AndrewMeadows**

### CODE REVIEW

File: interface/src/Application.cpp [hunk 1]
```diff
@@ -970,9 +970,7 @@ void Application::keyPressEvent(QKeyEvent* event) {
 
             case Qt::Key_E:
             case Qt::Key_PageUp:
-               if (!_myAvatar->getDriveKeys(UP)) {
-                    _myAvatar->jump();
-                }
+                _myAvatar->jump();
                 _myAvatar->setDriveKeys(UP, 1.0f);
                 break;
```
Comment (AndrewMeadows) [inline interface/src/Application.cpp]: The CharacterController class owned by MyAvatar is smart enough to not jump twice in a row.  In fact, it monitors failing jump() events for indication that the user is holding down the E key and really wants to jump into hover mode.  Hence we don't want to protect this call.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 245729cba8eb81827c57c2c844b360013837bbb3 available at: https://gist.github.com/58c0e51287bff3aa4bf2<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4573.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4573.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 0249b8c9c5927fcd52d5cdc4fe059335f1bba7e0 available at: https://gist.github.com/9ea38880495e9e9991c3<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4573.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4573.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: b37384d7590726c49773ec38a2cc2665ef10cc21 available at: https://gist.github.com/58c0e51287bff3aa4bf2<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4573.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4573.dmg

**Comment by AndrewMeadows** [comment]
A behavior change here was to let the avatar stay in "hover" when over the floor, but if a DOWN button is pressed it would transition to walk if the character's feet are within 2 meters of the floor.  However that is probably too much change for this PR so I'll pull that stuff out.

**Comment by PhilipRosedale** [comment]
I think the behavior should be:

Pressing 'Up' once makes you jump up about 1 meter into the air and then fall down again.  

Pressing/holding 'Up' while you are still in the air from jumping makes you steadily accelerate upward against gravity. 

Once you are more than 3 meters above an object, you no longer fall if you release the Up key - you hover.

**Comment by AndrewMeadows** [comment]
Ok, avatar jumps a little lower, hover transitions to fall when floor is within 3 meters of feet, and holding the UP key will eventually allow the avatar to fly up beyond the 3-meter floor detection threshold.

**Comment by hifi-gustavo** [comment]
Build console output for commit: fc4e6e93552939ef0d2cfd1ff3ed707485e2e381 available at: https://gist.github.com/58c0e51287bff3aa4bf2<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4573.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4573.dmg

**Comment by jherico** [comment]
Testing on my box

**Pull request merged**